DataConnectors/TrendMicroTippingPoint.json

```diff
@@ -0,0 +1,126 @@
+{
+    "id": "TrendMicroTippingPoint",
+    "title": "Trend Micro TippingPoint",
+    "publisher": "Trend Micro",
+    "descriptionMarkdown": "The Trend Micro TippingPoint connector allows you to easily connect your TippingPoint SMS IPS events with Azure Sentinel, to view dashboards, create custom alerts, and improve investigation. This gives you more insight into your organization's networks/systems and improves your security operation capabilities.",
+    "additionalRequirementBanner": "These queries and workbooks are dependent on Kusto functions to work as expected. Follow the steps to use the Kusto functions alias \"TrendMicroTippingPoint\" \nin queries and workbooks. [Follow these steps to get this Kusto function.](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/TrendMicro/TrendMicroTippingPoint)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "TrendMicroTippingPoint",
+            "baseQuery": "\nTrendMicroTippingPoint\n"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description": "TippingPoint IPS Events",
+            "query": "\nTrendMicroTippingPoint\n\n            | sort by TimeGenerated"
+        },
+        {
+            "description": "Top IPS Events",
+            "query": "\nTrendMicroTippingPoint\n\n            | summarize EventCountTotal = sum(EventCount) by DeviceEventClassID, Activity, SimplifiedDeviceAction\n            | sort by EventCountTotal desc"
+        },
+        {
+            "description": "Top Source IP for IPS Events",
+            "query": "\nTrendMicroTippingPoint\n\n            | summarize EventCountTotal = sum(EventCount) by SourceIP\n            | sort by EventCountTotal desc"
+        },
+        {
+            "description": "Top Destination IP for IPS Events",
+            "query": "\nTrendMicroTippingPoint\n\n            | summarize EventCountTotal = sum(EventCount) by DestinationIP\n            | sort by EventCountTotal desc"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "\nTrendMicroTippingPoint\n\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
```

Comment:
Please include datatype name, provider name and Appliance name to get appropriate results for connectors. Refer to the comment given for first sample query.